crates/turbopack-build/src/ecmascript/node/entry/runtime.rs

```diff
@@ -48,13 +48,16 @@ impl EcmascriptBuildNodeRuntimeChunk {
         };
 
         let mut code = CodeBuilder::default();
+        let output_root = output_root.to_string();
```

Comment:
Did you verify whether webpack uses the input path or the output paths to transform `import.meta.url`?

Reply:
https://github.com/vercel/next.js/blob/canary/packages/next/src/build/webpack/loaders/next-metadata-route-loader.ts#L219,

PR mimic / matches behavior to the existing dynamic metadata passes with test cases. afaik it's input, the original location of the source as it can be _any_ static assets can be read - i.e `readFileSync(path.fileurlToPath(import.meta.url)`